sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -0,0 +1,410 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#include <stdbool.h>
+#include <string.h>
+
+#include "net/gnrc/ipv6.h"
+#include "net/gnrc/ipv6/nib/conf.h"
+#include "net/gnrc/ipv6/nib/nc.h"
+#include "net/gnrc/ipv6/nib.h"
+#include "net/gnrc/netif.h"
+#include "random.h"
+
+#include "_nib-internal.h"
```

Comment:
Off-topic: but are these underscores (in file name and function names) a new convention? If yes: would have been nice if announced on the mailing list(s). If not: why here?